migrations/1640701224273-add-smart-tags.ts

```diff
@@ -0,0 +1,28 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class addSmartTags1640701224273 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      'INSERT INTO `permissions` (uuid, name) VALUES ("777d5c74-6201-4253-a56a-26d503e2abbd", "app:smart-filters")'
+    )
+
+    // Core user v1 keep access
+    await queryRunner.query(
+      'INSERT INTO `role_permissions` (role_uuid, permission_uuid) VALUES ("bde42e26-628c-44e6-9d76-21b08954b0bf", "777d5c74-6201-4253-a56a-26d503e2abbd")'
```

Comment:
Updated with core v1 role,

I've used the role from:
https://github.com/standardnotes/auth/blob/9dff11bd13a6f9d1fce4787de7b5b8de0a29e6a4/migrations/1624434102642-update_roles_and_permissions.ts#L14

@karolsojko I'm quite confident the whole thing makes sense, but  could you double check this part of the migration?

Reply:
That is correct - described more on Notion